docs_2.0/manual_EN/3.quick-start/4.nebua-graph-crud.md

```diff
@@ -0,0 +1,251 @@
+    +-----------+-------+--------+--------------+---------------------+------------------------+
+    | storaged2 | 44500 | ONLINE | 1            | nba:5               | nba:5                  |
+    +-----------+-------+--------+--------------+---------------------+------------------------+
+    Got 3 rows (time spent 744/1123 us)
+    ```
+
+    If the **Leader distribution** is uneven, use `BALANCE LEADER` to redistribute the partitions. For more information, see [BALANCE](TODO) [TODO].
+
+3. Use the `nba` graph space.
+
+    ```nGQL
+    (root@nebula) [(none)]> USE nba;
+    Execution succeeded (time spent 1322/2206 us)
+    ```
+
+    You can use `SHOW SPACES` to check the graph space you created.
+
+    ```nGQL
+    (root@nebula) [nba]> SHOW SPACES;
+    +------+
+    | Name |
+    +------+
+    | nba  |
+    +------+
+    Got 1 rows (time spent 1235/1934 us)
+    ```
+
+## Create tags and edge types
+
+### nGQL Syntax
+
+```nGQL
+CREATE {TAG | EDGE} {<tag name> | <edge type>}(<property name 1> <data type>, [<property name 2> <data type>, ...]);
+```
+
+### Demonstration
+
+Create tags `player` and `team`, edge types `follow` and `serve`.
+
+| Component name | Type      | Property                         |
+| -------------- | --------- | -------------------------------- |  |
+| player         | Tag       | name (string), age (int)         |
+| team           | Tag       | name (string)                    |
+| follow         | Edge type | degree (int)                     |
+| serve          | Edge type | start_year (int), end_year (int) |
+
+```nGQL
+(root@nebula) [nba]> CREATE TAG player(name string, age int);
+Execution succeeded (time spent 2694/3116 us)
+
+Thu, 15 Oct 2020 06:22:29 UTC
+
+(root@nebula) [nba]> CREATE TAG team(name string);
+Execution succeeded (time spent 2630/3002 us)
+
+Thu, 15 Oct 2020 06:22:37 UTC
+
+(root@nebula) [nba]> CREATE EDGE follow(degree int);
+Execution succeeded (time spent 3087/3467 us)
+
+Thu, 15 Oct 2020 06:22:43 UTC
+
+(root@nebula) [nba]> CREATE EDGE serve(start_year int, end_year int);
+Execution succeeded (time spent 2645/3123 us)
+
+Thu, 15 Oct 2020 06:22:50 UTC
+```
+
+## Create indexes
+
+Nebula Graph supports two different kinds of indexing to speed up query processing: tag indexes and edge type indexes. Normally, an index is created at the time the tag or edge type is created. For more information, see [Nebula Graph indexes](TODO) [TODO].
+
+> **CAUTION**: Creating indexes decreases the write performance.
+
+Indexes do not affect the data created before them. For example, if you created a vertex with the tag `player` and property `name`, and the value of `name` was `Tom`. Then you created an index based on the property `name`, this index were not able to cover vertex `Tom`. At this time, you need to rebuild the index to make it take effect on pre-existing data. For more information, see [Rebuild indexes](TODO) [TODO].
```

Comment:
this is a caution too.
I don't think this example is necessary.
Just caution & clarify:  make sure to rebuild index for pre-existing data. otherwise, the pre-existing data can't be accessed by the indices.